### PR TITLE
:arrow_up::one::two: pos_orderline_absolute_discount

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

File: pos_orderline_absolute_discount/static/description/index.html [hunk 1]
```diff
@@ -175,25 +175,25 @@ <h2>Need our service?</h2>
...
+                            <!--color: #75526b;-->
+                            <!--border: 3px dashed #75526b;-->
+                            <!--float: left;-->
+                            <!--padding: 4px 12px;-->
+                            <!-- -webkit-transform: rotate(-7deg);-->
+                            <!-- -o-transform: rotate(-7deg);-->
+                            <!-- -moz-transform: rotate(-7deg);-->
+                            <!-- -ms-transform: rotate(-7deg);">-->
+                    <!--Tested on Odoo<br/>12.0 enterprise-->
+                <!--</div>-->
```
Comment (yelizariev) [inline pos_orderline_absolute_discount/static/description/index.html]: Don't update each line

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_debt_notebook/static/description/index.html [hunk 2]
```diff
@@ -607,23 +607,23 @@ <h2>Need our service?</h2>
...
+                   color: #75526b;
+                   border: 3px dashed #75526b;
+                   float: left;
+                   padding: 4px 12px;
+                    -webkit-transform: rotate(-7deg);
+                    -o-transform: rotate(-7deg);
+                    -moz-transform: rotate(-7deg);
+                    -ms-transform: rotate(-7deg);">
+                     Tested on Odoo<br/>12.0 enterprise
+                </div>-->
```
Comment (yelizariev) [inline pos_debt_notebook/static/description/index.html]: No need to change it

File: pos_orderline_absolute_discount/static/description/index.html (hunk 1 above)
Comment (GabbasovDinar) [reply pos_orderline_absolute_discount/static/description/index.html]: DONE

File: pos_orderline_absolute_discount/static/description/index.html [hunk 3]
```diff
@@ -188,12 +188,12 @@ <h2>Need our service?</h2>
...
-                            -webkit-transform: rotate(-7deg);
-                            -o-transform: rotate(-7deg);
-                            -moz-transform: rotate(-7deg);
-                            -ms-transform: rotate(-7deg);">
-                    Tested on Odoo<br/>10.0 enterprise
-                </div>
+                             -webkit-transform: rotate(-7deg);
+                             -o-transform: rotate(-7deg);
+                             -moz-transform: rotate(-7deg);
+                             -ms-transform: rotate(-7deg);">
```
Comment (yelizariev) [inline pos_orderline_absolute_discount/static/description/index.html]: Thanks, but spaces are not needed

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_orderline_absolute_discount/static/description/index.html (hunk 3 above)
Comment (GabbasovDinar) [reply pos_orderline_absolute_discount/static/description/index.html]: @yelizariev DONE

**Review by yelizariev — APPROVED** [review]
Code LGTM

**Pull request merged**